packages/clients/src/api/container/index.ts

```diff
@@ -1 +1 @@
-export * as v1beta1 from './v1beta1'
```

Comment:
this file should be removed  or we don't have export inside baremetal ? @Vinzius